lit/SwiftREPL/FoundationTypes.test

```diff
@@ -41,3 +41,6 @@ n
 // CHECK:      {{.*R.*}}: Notification = {
 // CHECK-NEXT:   name = "abc"
 // CHECK-NEXT:   object = 42
+
+let tinky : Optional<UUID> = nil
```

Comment:
I ended up finding a super easy test case, incidentally.

Reply:
This depends on UUID being resilient, but it's better than nothing.

Reply:
I can write something more elaborate and probably robust if you want, but my attempt to understand how to write a makefile failed (if you want to explain me the magic, happy to hear).